runners/flink/src/main/java/org/apache/beam/runners/flink/translation/functions/FlinkExecutableStageFunction.java

```diff
@@ -163,7 +194,70 @@ public void accept(WindowedValue<?> input) throws Exception {
         SdkHarnessClient.RemoteOutputReceiver<?>> receiverMap =
         receiverBuilder.build();
 
-    try (SdkHarnessClient.ActiveBundle<InputT> bundle = processor.newBundle(receiverMap)) {
+    StateRequestHandler stateRequestHandler = new StateRequestHandler() {
```

Comment:
This class definitely is definitely big enough that it deserves is own (at least) static nested class. If you don't want to make that change here, then please add a TODO.